### PR TITLE
Update modifiers to call public view functions

### LINKED ISSUE #1179: Make modifiers call boolean functions
Most of our modifiers just add a `require` statement before the modified function is called.

For these cases, the modifier should be named like `onlyCondition`, for example, `onlyOwner`. This modifier should call a function that returns a boolean named like `isCondition`, for example, `isOwner`.

### PR DESCRIPTION
**Pull request by elopio**
Fixes #1179.
Requires #1268, #1269, #1270.

# 🚀 Description

It should be possible to query the condition that modifiers require.

- [x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [x] ✅ I've added tests where applicable to test my new functionality.
- [x] 📖 I've made sure that my contracts are well-documented.
- [x] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

### CODE REVIEW

File: contracts/crowdsale/validation/TimedCrowdsale.sol [hunk 1]
```diff
@@ -19,7 +19,7 @@ contract TimedCrowdsale is Crowdsale {
    */
   modifier onlyWhileOpen {
     // solium-disable-next-line security/no-block-members
```
Comment (frangio) [inline contracts/crowdsale/validation/TimedCrowdsale.sol]: We can remove this Solium annotation because we moved the expression to a function.

Comment (elopio) [reply contracts/crowdsale/validation/TimedCrowdsale.sol]: Removed.

**Comment by elopio** [comment]
@frangio yes, I've checked them all.
The first thing I noticed is that we don't have as many as I though.
And the second is that this is the most common pattern:

```
  modifier onlyRole(string _role)                                                                                                                                            
  {                                                                                                                                                                          
    checkRole(msg.sender, _role);                                                                                                                                            
    _;                                                                                                                                                                       
  }   
```
I think I would like to remove of all the `check` functions. Call the `require(hasRole)` directly on the modifiers. But this is different from what we discussed, and needs more thought.

**Comment by frangio** [comment]
Rebased on master.

**Comment by nventuro** [comment]
Yup, from what I recall `check` was only present in `Roles`, and I removed it for the reasons described above.

**Comment by frangio** [comment]
Thanks @elopio!

**Pull request merged**